src/DssSpell.t.base.sol

```diff
@@ -1146,41 +1086,41 @@ contract GoerliDssSpellTestBase is Config, Test {
         assertEq(clip.wards(address(end)), 1);
 
         // Check toll in/out
-        assertEq(psm.tin(), tin, concat("Incorrect-tin-", _ilk));
-        assertEq(psm.tout(), tout, concat("Incorrect-tout-", _ilk));
+        assertEq(psm.tin(), tin * WAD / 10000, _concat("Incorrect-tin-", _ilk));
```

Comment:
maybe consider computing the tin/tout rate from bps at the beginning to avoid redundancy

Reply:
Done. @iamchrissmith can I get a new approval please?